pkgs/development/tools/misc/texinfo/default.nix

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
Name this file `packages.nix` instead.

Reply:
Out of interest can you provide more context on this please, @oxij?

Reply:
This is another common convention of `nixpkgs`:
```
$ find pkgs -iname 'packages.nix'
```